a11y-meta-display-guide/2.0/draft/techniques/epub-metadata/index.html

```diff
@@ -219,122 +219,187 @@ <h3>Check for Node</h3>
+                        <dd>
+                            <p>If true it indicates that the <i>accessModeSufficient="auditory"</i> (all main content is provided in audio form) is present in the package document, otherwise if false it means that the metadata is not present.</p>
+                        </dd>
+                        <dt><var>synchronised_pre_recorded_audio</var></dt>
+                        <dd>
+                            <p>If true it indicates that the <i>accessibilityFeature="sychronizedAudioText"</i> is present in the package document, otherwise, if false, it means that the metadata is not present.</p>
+
+                            <p>This indicates that text-synchronised prerecorded audio narration (natural or synthesized voice) is included for substantially all textual matter, including all alternative descriptions, e.g. via a SMIL media overlay.</p>
+                        </dd>
+
+                        <dt><var>audio_content</var></dt>
+                        <dd>
+                            <p>If true it indicates that the <i>accessMode="auditory"</i> is present in the package document, otherwise, if false, it means that the metadata is not present.</p>
+
+                            <p>This indicates that prerecorded audio content is included as part of the work.</p>
+                        </dd>
+
+
+                    </dl>
+                    <h5>Variables setup</h5>
+                    <ol class="condition">
+                        <li><b>LET</b> <var>package_document</var> be the result of calling <a href="#pre-processing">preprocessing</a> given <var>package_document_as_text</var>.</li>
+
+                        <li><b>LET</b> <var>all_content_audio</var> be the result of calling <a href="#check-for-node">check for node</a> on <var>package_document</var>, <code class="xpath">/package/metadata/meta[@property="schema:<i>accessModeSufficient</i>" and normalize-space()="<i>auditory</i>"]</code>.</li>
+
+                        <li><b>LET</b> <var>synchronised_pre_recorded_audio</var> be the result of calling <a href="#check-for-node">check for node</a> on <var>package_document</var>, <code class="xpath">/package/metadata/meta[@property="schema:<i>accessibilityFeature</i>" and normalize-space()="<i>sychronizedAudioText</i>"]</code>.</li>
+
+                        <li><b>LET</b> <var>audio_content</var> be the result of calling <a href="#check-for-node">check for node</a> on <var>package_document</var>, <code class="xpath">/package/metadata/meta[@property="schema:<i>accessMode</i>" and normalize-space()="<i>auditory</i>"]</code>.</li>
+
+                    </ol>
+                    <h5>Instructions</h5>
+                    <ol class="condition">
+                        <li>
+                            <span><b>IF</b> <var>all_content_audio</var>:</span>
+                            <span><b>THEN</b> display <code id="ways-of-reading-prerecorded-audio-only">"Audio only"</code>.</span>
```

Comment:
Should this string be "Prerecorded audio only" now that the heading is removed?

Reply:
Oh good point if we mimic the "Additional Information of (Adaptation & Clarity) we still have each of those sub-headings, which maybe begs the question do we combine those and just have 1 heading there as well for "Additional Information"?

Reply:
Ok, I think that's what I need to do then and remove the sub-headings for Adaptation & Clarity and then improve the text strings adding more detail like you suggested Matt.